README.md

```diff
@@ -7,7 +7,7 @@ This codebase contains
 * Initial code for a TorQ on Managed kdb Insights application
```

Comment:
Should it read "or TorQ on 'a' Managed.."

Reply:
Not quite, no. This line is meant to say the application as a whole, not just FinSpace application (full name is technically "TorQ for Amazon FinSpace with Managed kdb Insights". I'll update it to be clearer.